hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionHealthProblem.java

```diff
@@ -0,0 +1,116 @@
+package pl.allegro.tech.hermes.api;
+
+public enum SubscriptionHealthProblem {
```

Comment:
I'm sure we can work something out but we have to map label to implementation somewhere. I like to keep these two close if possible.